apps/profil-api/src/main/java/no/nav/registre/testnorge/profil/consumer/command/GetProfileImageCommand.java

```diff
@@ -17,18 +17,24 @@
 
     @Override
     public byte[] call() {
-        return webClient
-                .get()
-                .uri(builder -> builder.path("/v1.0/me/photos/240x240/$value").build())
-                .header(HttpHeaders.AUTHORIZATION, "Bearer " + accessToken)
-                .retrieve()
-                .onStatus(
-                        HttpStatus::isError,
-                        clientResponse -> clientResponse
-                                .bodyToMono(String.class)
-                                .map(IllegalStateException::new)
-                )
-                .bodyToMono(byte[].class)
-                .block();
+
+        try {
+            return webClient
+                    .get()
+                    .uri(builder -> builder.path("/v1.0/me/photos/240x240/$value").build())
+                    .header(HttpHeaders.AUTHORIZATION, "Bearer " + accessToken)
+                    .retrieve()
+                    .onStatus(
+                            HttpStatus::isError,
+                            clientResponse -> clientResponse
+                                    .bodyToMono(String.class)
+                                    .map(IllegalStateException::new)
+                    )
+                    .bodyToMono(byte[].class)
+                    .block();
+        } catch (IllegalStateException e) {
+            log.warn("Bilde for bruker ikke funnet, melding: ", e);
+            return new byte[0];
```

Comment:
Hadde det vært bedre å returnere no content i stede for 0 bytes? også bare retunere optional her?

Reply:
Ja lurt, var ikke helt sikker på god løsning der, skal fikse 😄

Reply:
Jeg er uening med meg selv. Kan du retuenre 404? Så håntere det i frontend?